src/components/question-editor/QuestionEditor.tsx

```diff
@@ -216,36 +257,32 @@ const QuestionEditor: FC<QuestionEditorProps> = ({
 
       {isSingleChoice && <RadioGroup sx={styles.group}>{options}</RadioGroup>}
 
-      {!isOpenAnswer && (
-        <Box
-          data-testid='addNewAnswerBtn'
-          onClick={addNewOneAnswer}
-          sx={styles.addRadio(isEmptyAnswer)}
-        >
+      {isOpenAnswer && openAnswerOptions}
+
+      <Box
+        data-testid='addNewAnswerBtn'
+        onClick={addNewOneAnswer}
+        sx={
+          isOpenAnswer
+            ? spliceSx(styles.addRadio(isEmptyAnswer), styles.openAnswer)
+            : styles.addRadio(isEmptyAnswer)
+        }
+      >
+        {isOpenAnswer ? (
+          <Typography color={isEmptyAnswer ? 'primary.300' : ''}>
```

Comment:
Is `color = ''` exists?
```suggestion
          <Typography color={isEmptyAnswer ? 'primary.300' : undefined}>
```